packages/imperative/src/ConfigurationLoader.ts

```diff
@@ -55,7 +55,9 @@ export class ConfigurationLoader {
         // override the config with the content of the module specified
         if (config.configurationModule != null) {
             try {
+                const daemonMode = config.daemonMode;
                 config = callerFileRequirer(config.configurationModule);
+                config.daemonMode = daemonMode;
```

Comment:
it looks like `daemonMode` piece is intentionally not overridden - is that correct?

Reply:
That is correct. We decided we wanted to keep daemonMode because it is not always in use, which means we couldn't set it to be true in the Imperative configuration in the CLI. The CLI, at startup, will search for `--daemon` in its list of arguments, and if the argument is present, daemonMode is set to true. This will suppress any Config related errors (bad secure storage mechanism, invalid JSON, etc) at server startup time only.